static/app/views/monitors/components/overviewTimeline/useTimeWindowConfig.tsx

```diff
@@ -0,0 +1,15 @@
+import {useMonitorDates} from './useMonitorDates';
+import {getConfigFromTimeRange} from './utils';
+
+interface Options {
+  /**
+   * The width of the timeline influences how we calculate the rollup value
+   */
+  timelineWidth: number;
+}
+
+export function useTimewindowConfig({timelineWidth}: Options) {
```

Comment:
nit: can we capitalize W here `useTimeWindowConfig`? to match file/variable name

Reply:
Yeah, let me fix this up